modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts

```diff
@@ -1,21 +1,31 @@
 import {load} from '@loaders.gl/core';
 import {getSupportedGPUTextureFormats, selectSupportedBasisFormat} from '@loaders.gl/textures';
-import {I3sTilesetHeader, NodePage} from '../../types';
 import {I3SNodePageLoader} from '../../i3s-node-page-loader';
 import {normalizeTileNonUrlData} from '../parsers/parse-i3s';
 import {getUrlWithToken, generateTilesetAttributeUrls} from '../utils/url-utils';
+import type {LoaderOptions} from '@loaders.gl/loader-utils';
+import {
+  I3STilesetHeader,
+  LodSelection,
+  NodePage,
+  NodeInPage,
+  Obb,
+  meshGeometry,
+  meshMaterial,
+  I3SMaterialDefinition
+} from '../../types';
 
 /**
  * class I3SNodePagesTiles - loads nodePages and form i3s tiles from them
  */
 export default class I3SNodePagesTiles {
-  tileset: I3sTilesetHeader;
+  tileset: I3STilesetHeader;
   nodePages: NodePage[] = [];
   pendingNodePages: {promise: Promise<NodePage>; status: 'Pending' | 'Done'}[] = [];
   nodesPerPage: number;
-  options: {[key: string]: any};
-  lodSelectionMetricType: any;
-  textureDefinitionsSelectedFormats: any[] = [];
+  options: LoaderOptions;
```

Comment:
Nice. For a future PR you may be able to define I3SLoaderOptions by extending LoaderOptions...